common-rest/src/main/java/org/uniprot/api/rest/download/queue/BaseAbstractMessageListener.java

```diff
@@ -36,16 +36,19 @@ public abstract class BaseAbstractMessageListener implements MessageListener {
     private final DownloadJobRepository jobRepository;
 
     protected final RabbitTemplate rabbitTemplate;
+    private final HeartBeatProducer heartBeatProducer;
 
     public BaseAbstractMessageListener(
             DownloadConfigProperties downloadConfigProperties,
```

Comment:
Here DownloadConfigProperties and AsyncDownloadQueueConfigProperties are used in the same context. Do you think we could merge both? because I saw in some other places that they are used independently.

Reply:
I think HeartBeatConfigs are used independently.  And even DownloadConfigProperties refers to the file configuration part of that and not the placeholder for whole configurations.